File: app/src/main/java/com/pnuema/bible/ui/viewholders/CopyrightViewHolder.java
package com.pnuema.bible.ui.viewholders;

import androidx.annotation.NonNull;
import androidx.recyclerview.widget.RecyclerView;
import android.view.LayoutInflater;
import android.view.ViewGroup;
import android.widget.TextView;

import com.pnuema.bible.R;
import com.pnuema.bible.statics.HtmlUtils;

public class CopyrightViewHolder extends RecyclerView.ViewHolder {
    private TextView copyrightText;

    public CopyrightViewHolder(@NonNull ViewGroup parent) {
        super(LayoutInflater.from(parent.getContext()).inflate(getLayout(), parent, false));

        copyrightText = itemView.findViewById(R.id.copyrightText);
    }

    public void setText(String text) {
        copyrightText.setText(HtmlUtils.fromHtml(text));
    }

    public static int getType() {
        return getLayout();
    }

    private static int getLayout(){
        return R.layout.listitem_copyright;
    }
}

File: app/src/main/java/com/pnuema/bible/android/ui/viewholders/CopyrightViewHolder.kt
package com.pnuema.bible.android.ui.viewholders

import android.view.LayoutInflater
import android.view.ViewGroup
import androidx.recyclerview.widget.RecyclerView
import com.pnuema.bible.android.R
import com.pnuema.bible.android.databinding.ListitemCopyrightBinding
import com.pnuema.bible.android.statics.HtmlUtils

class CopyrightViewHolder(
        parent: ViewGroup,
        private val binding: ListitemCopyrightBinding = ListitemCopyrightBinding.inflate(LayoutInflater.from(parent.context), parent, false)
) : RecyclerView.ViewHolder(binding.root) {
    companion object {

        val type: Int
            get() = layout

        private val layout: Int
            get() = R.layout.listitem_copyright
    }

    fun bind(text: String) {
        binding.copyrightText.text = HtmlUtils.fromHtml(text)
    }
}
